src/MentorBot.Functions/Connectors/OpenAirConnector.cs

```diff
@@ -141,6 +141,11 @@ public async Task SyncUsersAsync()
         {
             var storedUsers = await _storageService.GetAllUsersAsync();
             var openAirModelUsers = await _client.GetAllUsersAsync();
+            foreach (var user in openAirModelUsers)
+            {
+                user.Name = user.Name.ToLower();
```

Comment:
Why are you doing this as a separate foreach when there is a openAirModelUsers foreach in line 154?
Add unit test.